tutorials/plot_receptive_field.py

```diff
@@ -190,7 +189,7 @@
 scores = np.zeros_like(alphas)
 models = []
 for ii, alpha in enumerate(alphas):
-    rf = ReceptiveField(tmin, tmax, sfreq, freqs, estimator=Ridge(alpha))
+    rf = ReceptiveField(tmin, tmax, sfreq, freqs, estimator=alpha)
```

Comment:
well I think in the name of our conversations about keeping the tutorials / examples clearly delineated from one another, it should be in the tutorial. Unless we had one very specific and short example that made a comparison between the two.